client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/rates-step/shipping-rate.js

```diff
@@ -81,11 +92,11 @@ class ShippingRate extends Component {
 						<div className="rates-step__shipping-rate-description-title">{ title }</div>
 						<div className="rates-step__shipping-rate-description-details">
 							{ details }
-							{ null !== requiredSignatureCost && requiredSignatureCost > 0 ? (
+							{ null !== requiredSignatureCost && signatureRequirementAllowed ? (
 								<CheckboxControl
 									label={ translate(
-										'Signature Required (+%(price)s)',
-										{ args: { price: formatCurrency( requiredSignatureCost, 'USD') } }
```

Comment:
I am assuming the formatCurrency is removed since the cost is coming back in the correct currency, and we shouldn't hard code it all to USD?

Reply:
I moved the price string generation up here: https://github.com/Automattic/woocommerce-services/pull/1786/files#diff-464e08994433b72877e47f9464d6d925R61 which still uses `formatCurrency` with `USD` specified

Reply:
Ah i don't know how i didn't see that, thanks!